.pre-commit-config.yaml

```
@@ -62,6 +62,8 @@ repos:
     rev: "v0.0.261"
     hooks:
       - id: ruff
+        args:
+          - --fix
```

As per [Ruff's docs](https://beta.ruff.rs/docs/usage/#pre-commit), you will need another flag.
```suggestion
          - --fix
          - --exit-non-zero-on-fix
```

---

Not really, pre-commit already fails hooks if any files change.

---

Ah, I wonder why Ruff wants that flag then.

---

I had the same assumption until I tried it myself locally when writing a local hook last week. I think it is a reasonable but incorrect assumption that the Ruff documentation writers have made :slightly_smiling_face: